### PR TITLE
German translation for 3.5

### PR DESCRIPTION
**Pull request by cornelinux**

### CODE REVIEW

**Review by plettich — CHANGES_REQUESTED** [review]
The German translation for the server (`privacyidea/translations/de/LC_MESSAGES/messages.po`) is still missing.

**Comment by cornelinux** [comment]
@plettich thanks. Forgot to add it.

File: privacyidea/translations/de/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -0,0 +1,2717 @@
...
+msgid "Set the length of registration codes."
+msgstr "Setzt die Länge des Registrierungscodes."
+
+#: lib/policy.py:1979
+msgid ""
+"Specify the required contents of the registration code. (c)haracters, (n)umeric, "
+"(s)pecial. Use modifiers +/- or a list of allowed characters [1234567890]"
+msgstr ""
+"Definiert den erforderlichen Inhalt des Registrierungscodes. Zeichen (c), "
+"Nummern (n) und Spezialzeichen (s). Die Modifikatoren +/- können verwendet "
```
Comment (plettich) [inline privacyidea/translations/de/LC_MESSAGES/messages.po:637]: shouldn't this be "Sonderzeichen"? I think this is the common term.

**Review by plettich — COMMENTED** [review]
just a minor spelling thing. I didn't check all the translations... Looks good otherwise.

File: privacyidea/translations/de/LC_MESSAGES/messages.po (hunk 1 above)
Comment (cornelinux) [reply privacyidea/translations/de/LC_MESSAGES/messages.po:637]: You are totally correct (aka right) ;-)

Comment (cornelinux) [reply privacyidea/translations/de/LC_MESSAGES/messages.po]: Done in ee9d9bff

**Review by plettich — APPROVED** [review]
looks good

**Pull request merged**